README.md

```diff
@@ -37,6 +37,8 @@ VMware vSphere is a cloud computing virtualization platform. The vSphere module
 
 ### Installing the vSphere module
 
+The following are *dependencies* of the module and as such should be installed on the system from which the module is configured to be used on i.e. in a master-agent setup, the *agent* is the one that requires these dependencies installed.
```

Comment:
Looks good, but maybe reword along the lines of:
"The following are *dependencies* of the module. Install these on the system which you configure the module on. For example, in a master-agent setup, install the dependencies on the agent".

Reply:
Updated.